Apromore-Core-Components/Apromore-Manager/src/main/java/org/apromore/util/datetime/DurationUnit.java

```diff
@@ -0,0 +1,100 @@
+/*-
+ * #%L
+ * This file is part of "Apromore Core".
+ * %%
+ * Copyright (C) 2018 - 2020 Apromore Pty Ltd.
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ *
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * #L%
+ */
+/**
+ * Duration unit
+ */
+
+package org.apromore.util.datetime;
+
+import java.util.Map;
+import java.util.HashMap;
+import java.time.temporal.ChronoUnit;
+
+public final class DurationUnit {
+
```

Comment:
Please move the comment "/** Duration unit...." to above the class DurationUnit, rather than for the package. The package will be repeated for each class file, the comment is for each class, not for the package.